src/design/service-design/service-design-approach.md

```diff
@@ -1,9 +1,8 @@
 ---
 title: Service design approach
-redirect_from:
-  - /guides/service-design-at-dxw
-last_reviewed_at: ""
```

Comment:
`last_reviewed_at` should stay and the `redirect_from` should move to the new index